### DISCUSSION TITLE
"ADT needs to validate subscription details.  X days remaining." warning encountered

### DISCUSSION THREAD

**Discussion by Tiuser4567** [airsdk/Adobe-Runtime-Support#2335] [Q&A]
Hello, we purchased a license last month (Nov 4) and all was well and the harman splash screen is removed.
However, while building the APK file this week, we suddenly encountered the ff warning below

```
WARNING: ADT license file for developer ID "xxxxxxx" reported as: ERROR
     [java] License file has been removed: please contact adobe.support@harman.com to renew your license
     [java] WARNING - ADT needs to validate subscription details. 0 days remaining
      [java] WARNING: could not generate license file for APK, splash screen will be injected. Please contact adobe.support@harman.com for support
```

Verified that our license is still valid as putting the details in https://airsdk.harman.com/renew results to 
"As your AIR SDK license is still 'active', no need to renew (License expiration date is beyond 30 days)."

Our build server is within an internal network (and no access to the internet) 
Does above warning means that we need to connect the build server to have internet access to validate the license?
If yes, any other way to be able to do some "offline" validation or exception? 
(as unfortunately the process to connect the build server will go through several people and will take some days)

**Comment by ajwfrost** [comment]
Yes if it's an offline system then you're hitting the problem where we're trying to validate the license file.. can you please contact the adobe support email address and explain this scenario (or just put "FAO Andrew" and link to this issue..) - we can then get an exception set up.

thanks
